utils/queries.js

```diff
@@ -62,6 +62,10 @@ export const SIGNAL_STATUS_QUERY = {
       key: "limit",
       value: "99999999",
     },
+    {
+      key: "where",
+      value: "operation_text!='Communication issue'"
```

Comment:
other option is "operation_state!='3'", but I find this easier to remember

Reply:
nice—this looks good to me 🙏